### DISCUSSION TITLE
Add the Super key to the modifier key in the Mozc key binding settings

### DISCUSSION THREAD

**Discussion by jidaikobo-shibata** [google/mozc#872] [Ideas]
Mozc helps me every day. Thank you for your development.

The Mozc property key binding settings do not seem to accept the Super key as a modifier key. I would be happy if Super keys were accepted as well.

My environment is Ubuntu 22.04.3.
I tried with these keyboards
- Apple Aluminium (JIS) keyboard
- Japanese keyboard

Thanks.

![Mozc キー設定](https://github.com/google/mozc/assets/9050388/4e3e9347-f25f-43ba-a3bd-b406570eaf8d)
↑Mozc key Setting

![Mozc キーバインディング設定](https://github.com/google/mozc/assets/9050388/9e2eb941-068f-4247-b6da-bebaf7e9b17d)
↑Mozc Key binding Setting.  this window's input field does not accept Super key.

[ja]

毎日、Mozcに助けられています。開発、ありがとうございます。

Mozc プロパティのキーバインディング設定では、モディファイアキーとして、Superキーを受け付けてくれないようです。Superキーも受け付けてくれるようになると嬉しいです。

**Comment by hiroyuki-komatsu** [comment]
Hi jidaikobo-shibata,
Thank you for your feedback.

At this moment, Mozc supports only Shift and Control keys as modifier keys.
I think Alt and Super (including Windows / Command) keys are used for the operating system.

On the other hand, I'd like to know the use case of those modifier keys for IMEs.
If there are other IMEs that use Alt or Super, please let us know.

Best,

**Reply by jidaikobo-shibata** [reply]
Thank you for your reply.

You are right, and I think there are probably few examples of using the Super key (command / Windows) as a modifier key for IME operations. I wouldn't have thought about it unless I was in the current situation.

I moved from macOS to Ubuntu, but I can't get rid of the Apple Aluminum keyboard I've been using for decades, so I use it by swapping control and super in gsettings. By doing so, I can use the Apple Aluminum keyboard almost without any discomfort. For this reason, I thought I would be happy if I could make the Super key a modifier key for Mozc.

Regarding the alt key (option key), it cannot be registered as a new modifier key in Mozc, but it exists in Kotoeri's preset, as shown in the screenshot.

Thanks

![mozc key setting option(alt) is exists](https://github.com/google/mozc/assets/9050388/7d8338da-98c5-424d-b6a9-6d7422840c73)
